data-raw/simulate-data.R

```diff
@@ -152,37 +152,39 @@ insert_specific_atc <- function(data, proportion = 0.3) {
 # Insert a few cases where purchases of metformin are used for other purposes
 # than diabetes.
 insert_false_metformin <- function(data, proportion = 0.05) {
-  if (!all(colnames(data) %in% c("atc", "name"))) {
-    return(data)
-  }
-  data |>
-    dplyr::mutate(
-      atc = dplyr::if_else(
-        indo %in% c("0000092", "0000276", "0000781") & insertion_rate(proportion),
-        "A10BA02",
-        atc
-      ),
-      name = dplyr::if_else(
-        indo %in% c("0000092", "0000276", "0000781") & insertion_rate(proportion),
-        "metformin",
-        name
+  if (all(c("atc", "name", "indo") %in% colnames(data))) {
+    data |>
+      dplyr::mutate(
+        atc = dplyr::if_else(
+          indo %in% c("0000092", "0000276", "0000781") & insertion_rate(proportion),
+          "A10BA02",
+          atc
+        ),
+        name = dplyr::if_else(
+          indo %in% c("0000092", "0000276", "0000781") & insertion_rate(proportion),
+          "metformin",
+          name
+        )
       )
-    )
+  } else {
+  return(data)
```

Comment:
```suggestion
  data
```

Only minor thing here, since each if else condition outputs a tibble, we don't need to use return here.

Reply:
Ah, nice!